joomla/joomla-cms
# 3.9 dev

Pull Request for Issue #19432 .

### Summary of Changes
Adding the additional string " and consider to uncomment the line '# RewriteBase /'" to the tag 
"COM_CONFIG_FIELD_SEF_REWRITE_DESC"

### Testing Instructions
goto the backend and navigate to system > configuration. You will see the tab "site" activated. Move your mouse over the item "Use URL Rewriting".


### Expected result
The tooltip should include this longer substring:
Apache users only!
Rename htaccess.txt to .htaccess before activating and consider to uncomment the line '# RewriteBase /'.


### Actual result
The tooltip includes this substring:
Apache users only!
Rename htaccess.txt to .htaccess before activating.


### Documentation Changes Required
